frontend/README.md

```diff
@@ -0,0 +1,19 @@
+## This is a module to build frontend javascript projects
+
+At the moment it only contains "user profile page" bundle.
+
+To build it, just run
+
+```mvn install```
+
+Following mvn arguments is overrideable from command line:
```

Comment:
I suggest something like:
"The following Maven properties can be overridden on the command line with `-Dkey=value`:"